plugins/collections/README.md

```diff
@@ -53,3 +53,23 @@ Example payload:
     "pipelineIds": [12, 35, 47, 89]
 }
 ```
+
+#### Update a collection
+You can update the name, description, or pipelineIds of a collection.
+
+`PUT /collections/{id}`
+
+**Arguments**
+
+* `name` - An optional new name for the collection. Names must be unique.
+* `description` - An optional new description for the collection.
+* `pipelineIds` - An optional new array of ids of pipelines to be added to the collection.
```

Comment:
@ian-fox assuming you meant `{ pipelineIds: [3] }`, then collection 1 will have pipelines `[3]` and the other 2 get removed. Would it be better to defer adding / removing pipelines to a separate route and just not allow pipelineIds to be modified in the UPDATE route? My thought process was that since mostly only the UI is going to be interfacing with the API, then if a user wants to add a pipeline with id `3` to a collection and it already has pipelines `[1, 2]`, then the UI would make a request to `PUT /collections/1` with `{ pipelineIds: [1, 2, 3] }`.